### PR TITLE
Licences

### PR DESCRIPTION
**Pull request by ValdoTR**
Added the 3 licences for each part of the files:
* Code licence
* Map licence
* Assets licence

And reference them in a new section in the README of the project.

Also added the JSON properties `mapCopyright` and `tilesetCopyright`.

### CODE REVIEW

File: LICENCE.assets [hunk 1]
```diff
@@ -0,0 +1,21 @@
+ASSETS LICENCE
+
+
```
Comment (moufmouf) [inline LICENCE.assets:3]:
```suggestion
Tilesets distributed with this map are subject to different licences. The licences complete text is below.
The licence attributed to each tileset can be found in the "tilesetCopyright" property of each tileset in the map.
```

File: LICENCE.assets [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+3. These Assets or those modified pursuant to the previous item may undergo Distribution, etc. with maps produced for WorkAdventure. Furthermore, the User may not perform Distribution, etc. of individual Assets or those modified, or Distribution, etc. in combination with programs, etc. other than the map. However, the User may duplicate, transfer, publicly transmit, or enable transmission of personally-modified Assets to other Authorized Users free of charge.
+
+
+THIRD PARTIES ASSETS
+
+The User shall handle assets with third parties resources licence as follows:
+
+1. Follow the specific licence of each third party asset
+
+2. Credit the author with the "tilesetCopyright" property present in the properties of each tilesets in the map.json file
```
Comment (moufmouf) [inline LICENCE.assets:21]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -13,4 +13,12 @@ npm install
...
\ No newline at end of file
+The project will run at http://localhost:8080/
+
+## Licences
+
+This project contains multiple licences:
+
+* [Code licence](./LICENCE.code) *(all files except those for other licenses)*
+* [Map licence](./LICENCE.map) *(`map.json` and the map visual as well)*
+* [Assets licence](./LICENCE.assets) *(the files inside the `src/assets/` folder)*
```
Comment (moufmouf) [inline README.md:24]:
```suggestion
* [Assets licence](./LICENCE.assets) *(the files inside the `src/assets/` folder)*

## How to use this starter kit

If you add third party tilesets in your map, do not forget to:

1. Credit the author and license with the "tilesetCopyright" property present in the properties of each tilesets in the `map.json` file

2. Add the license text in LICENSE.assets
```

File: map.json [hunk 4]
```diff
@@ -230,6 +236,12 @@
          "imagewidth":320,
          "margin":0,
          "name":"tileset6_export",
+         "properties":[
+          {
+           "name":"tilesetCopyright",
+           "type":"string",
+           "value":"https:\/\/workadventu.re\/"
```
Comment (moufmouf) [inline map.json:243]: We should probably have 2 properties:

"copyright": "© 2020 Full Name <email>"
"license": "CC BY-SA 3.0"

Comment (ValdoTR) [reply map.json:243]: Ok, this is a new task.

Comment (moufmouf) [reply map.json:243]: At least, put:

"© 2021 WorkAdventure - Lincense: WORKADVENTURE SPECIFIC RESOURCES LICENSE (see LICENCE.assets file)"

Comment (ValdoTR) [reply map.json]: Good catch

**Pull request merged**